content/en/docs/getting-started/_index.md

```diff
@@ -32,9 +32,16 @@ apt-get install -y --no-install-recommends \
   unzip \
   uuid-runtime \
   python3 \
-  python3-setuptools
+  python3-setuptools \
+  python3-pip
 ```
 
+## Installing `kraft` on Fedora and other dnf-based-systems
```

Comment:
I would not add a title, since there is no separate title for the above command.
You can just rephrase the next phrase to make it an introduction.